Library/Formula/gobject-introspection.rb

```diff
@@ -43,6 +41,13 @@ def install
   end
 
   test do
-    system bin/"g-ir-annotation-tool", "--help"
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["libffi"].opt_lib/"pkgconfig"
+    system "git", "clone", "-b", "steptwo_gir",
+           "https://git.gitorious.org/gobject-introspection-tutorial/code.git"
```

Comment:
"System notice: Gitorious is being acquired by GitLab and gitorious.org will shut down end of May." The glib boilerplate is a lot to inline.